docs/pages/connect-your-client/model-context-protocol/database-access.mdx

```diff
@@ -46,7 +46,7 @@ or [Database Access Controls](../../enroll-resources/database-access/rbac.mdx).
 First, sign in into your Teleport cluster using `tsh login`:
 
 ```code
-$ tsh login --proxy=<Var name="teleport.example.com:443" /> --user=myuser@example.com
+$ tsh login --proxy=<Var name="teleport.example.com" />:443 --user=myuser@example.com
```

Comment:
The proxy isn't guaranteed to be on 443, so I think just drop it from here. Though I see that `teleport.example.com:443` is used throughout MCP docs as the value for the proxy.